### DISCUSSION TITLE
UML Support (Plantuml or mermaid-js)

### DISCUSSION THREAD

**Discussion by lupu60** [outline/outline#1990] [Ideas]
Add the option to render plantuml or mermaid.

## Expected Behavior: 
When using the plantuml syntax render the png not the code. 
```plantuml
Bob -> Alice : hello
Alice -> Bob : hi
```
like this 
![image](https://user-images.githubusercontent.com/5337610/112608326-7c432080-8e1a-11eb-865f-1d59a043d3c4.png)

Same behavior for mermaid-js.

**Comment by tommoor** [comment]
It would be cool, the place to add this feature would be in our open source editor here:

https://github.com/outline/rich-markdown-editor

**Reply by matbrgz** [reply]
mermaid-js is a more sophisticated project.
